components/life-cycle-extension/src/main/java/com/wso2telco/dep/lifecycleextension/util/Constants.java

```diff
@@ -0,0 +1,23 @@
+package com.wso2telco.dep.lifecycleextension.util;
+
+public class Constants {
+
+    public static final String STATE_CREATED="Created";
+    public static final String ADMIN_ROLE="admin";
+
+    public static final String CLAIM_EMAIL = "email";
+    public static final String CLAIM_GIVEN_NAME = "given_name";
+
+    public static final String WORKFLOW_PROPERTIES_XML_FILE = "workflow-configuration.xml";
+    public static final String KEY_WORKFLOW_EMAIL_NOTIFICATION_HOST = "workflow.email.notification.host";
+    public static final String KEY_WORKFLOW_EMAIL_NOTIFICATION_FROM_ADDRESS = "workflow.email.notification.from.address";
+    public static final String KEY_WORKFLOW_EMAIL_NOTIFICATION_FROM_PASSWORD = "workflow.email.notification.from.password";
+    public static final String SERVICE_HOST = "service.host";
+    public static final String PUBLISHER_ROLE_START_WITH ="workflow.Publisher.role.start.with";
+    public static final String PUBLISHER_ROLE_END_WITH ="workflow.Publisher.role.end.with";
+    public static final String MANDATE_SERVICE_HOST = "mandate.service.host";
+
```

Comment:
why PUBLISHER_ROLE_START_WITH, PUBLISHER_ROLE_END_WITH, and MANDATE_SERVICE_HOST properties?

Reply:
the same issue as WorkflowProperties.java will fix